### PR TITLE
Javascript/javascript1/week3

### PR DESCRIPTION
**Pull request by SlaviaF**

### CODE REVIEW

File: javascript/javascript1/week3/smartEase.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+//SAVE A NOTE
+const notes = [];
+
+
+function saveNote(content, id) {
+    const addNotes = { content: content, id: id }
+    notes.push(addNotes);
```
Comment (cameronjdavis) [inline javascript/javascript1/week3/smartEase.js]: Variables (and constants) are normally named like objects, for example `note`. The name `addNotes` makes it sound like a function that will do something. A name like `note` makes it sound like what it is, an object that holds information.
```suggestion
    const note = { content: content, id: id }
    notes.push(note);
```

File: javascript/javascript1/week3/smartEase.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+//Additional feature to check if the task has been completd.
+//PENDING NOTES
+
+function displayPendingNotes(id, done = false) {
+    for (i = 0; i < notes.length; i++) {
+        if (done && notes[i].id === id) {
+
+            console.log('Id ' + notes[i].id + ' has been completed');
+            break;
```
Comment (cameronjdavis) [inline javascript/javascript1/week3/smartEase.js]: If I call `displayPendingNotes(2, false);` will `console.log()` ever be called? What is the purpose of `done`?

File: javascript/javascript1/week3/cactus.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    if (value.length < 1) {
+        console.log("Add some activities before calling showStatus")
+    }
+
+    for (i = 0; i < value.length; i++) {
+        totalDuration = totalDuration + value[i].duration;
+
+    }
+    if (totalDuration > 400) {
+        console.log("You have reached your limit, no more smartphoning for you!");
```
Comment (cameronjdavis) [inline javascript/javascript1/week3/cactus.js:36]: Do you want to show the warning when `totalDuration` is 400 or more, or do you want to show it when `totalDuration` is 401 or more?
```suggestion
    if (totalDuration >= 400) {
        console.log("You have reached your limit, no more smartphoning for you!");
```

**Review by cameronjdavis — CHANGES_REQUESTED** [review]

File: javascript/javascript1/week3/cactus.js (hunk 3 above)
Comment (cameronjdavis) [reply javascript/javascript1/week3/cactus.js:36]: I read the homework again and see that it asks to check "usage is above the limit he set", so your original code is correct. :)

File: javascript/javascript1/week3/smartEase.js (hunk 1 above)
Comment (SlaviaF) [reply javascript/javascript1/week3/smartEase.js]: Yes...I will keep that in mind the next  and will change it now

File: javascript/javascript1/week3/smartEase.js (hunk 2 above)
Comment (SlaviaF) [reply javascript/javascript1/week3/smartEase.js]: Yess...IT doesnt make any sense. I will  remove this and try to think about something else

File: javascript/javascript1/week3/cactus.js (hunk 3 above)
Comment (SlaviaF) [reply javascript/javascript1/week3/cactus.js:36]: Thankyou for the feedback

Comment (cameronjdavis) [reply javascript/javascript1/week3/cactus.js:36]: No worries, Slavia. Good coding.

**Pull request merged**